tests/tensorflow/test_models/sequential_model.py

```diff
@@ -11,8 +11,8 @@
  limitations under the License.
 """
 
-from tensorflow.keras import Sequential, layers
-from tensorflow.keras.initializers import Constant  # pylint: disable=no-name-in-module
+from nncf.tensorflow.tf_internals import Sequential, layers
+from tensorflow.python.keras.initializers import Constant # pylint: disable=no-name-in-module
```

Comment:
Could you please explain why it was changed? It is a public API. This change should be reverted.
This code
```python
from tensorflow.keras.initializers import Constant
```
is correct. Please see [this](https://www.tensorflow.org/versions/r2.8/api_docs/python/tf/keras/initializers/Constant) for more details.

Reply:
I thought is was for compatibility, but `tf.keras.initializers` is right. Modified.